VimeoNetworkingExample-iOS/VimeoNetworkingExample-iOSTests/FileTransferTests.swift

```diff
@@ -0,0 +1,83 @@
+//
+//  Created by Nguyen, Van on 2/12/19.
+//  Copyright © 2019 Vimeo. All rights reserved.
+//
+//  Permission is hereby granted, free of charge, to any person obtaining a copy
+//  of this software and associated documentation files (the "Software"), to deal
+//  in the Software without restriction, including without limitation the rights
+//  to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+//  copies of the Software, and to permit persons to whom the Software is
+//  furnished to do so, subject to the following conditions:
+//
+//  The above copyright notice and this permission notice shall be included in
+//  all copies or substantial portions of the Software.
+//
+//  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+//  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+//  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+//  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+//  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+//  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+//  THE SOFTWARE.
+//
+
+import XCTest
+import OHHTTPStubs
+
+class FileTransferTests: XCTestCase {
+
+    override func setUp() {
+        
+        super.setUp()
+        
+        VimeoClient.configureSharedClient(withAppConfiguration: AppConfiguration(clientIdentifier: "{CLIENT_ID}",
+                                                                                 clientSecret: "{CLIENT_SECRET}",
+                                                                                 scopes: [.Public, .Private, .Purchased, .Create, .Edit, .Delete, .Interact, .Upload],
+                                                                                 keychainService: "com.vimeo.keychain_service",
+                                                                                 apiVersion: "3.3.10"), configureSessionManagerBlock: nil)
+        
+        stub(condition: isPath("/videos/" + Constants.CensoredId)) { _ in
+            let stubPath = OHPathForFile("clip.json", type(of: self))
+            return fixture(filePath: stubPath!, headers: ["Content-Type":"application/json"])
+        }
+    }
+
+    override func tearDown() {
+        
+        super.tearDown()
+        
+        OHHTTPStubs.removeAllStubs()
+    }
+
+    func testParsingFileTransferResponse() {
+        let request = Request<VIMVideo>(path: "/videos/" + Constants.CensoredId)
+        
+        let expectation = self.expectation(description: "Network call expectation")
+        
+        _ = VimeoClient.sharedClient.request(request) { response in
+            switch response
+            {
+            case .success(let result):
+                let video = result.model
+                
+                XCTAssertNotNil(video.fileTransfer, "\"fileTransfer\" response should have been not nil.")
```

Comment:
How about instead testing for equality with string  "www.google.com"?

Reply:
@nicolelehrer that assertion should be right below this one :)

Reply:
Ah ok - I thought this was testing `file.transfer.link` at first glance, but I see this is just the object. I could see this not being necessary if we're inspecting the object below to get the URL but your call.